internal/server/data/organization.go

```diff
@@ -158,3 +158,28 @@ func UpdateOrganization(tx WriteTxn, org *models.Organization) error {
 func CountOrganizations(tx ReadTxn) (int64, error) {
 	return countRows(tx, organizationsTable{})
 }
+
+type ForgottenDomain struct {
+	OrganizationName   string
+	OrganizationDomain string
+	// LastSeenAt is the last time the user logged into this organization.
+	LastSeenAt time.Time
+}
+
+func GetForgottenDomainsForEmail(tx ReadTxn, email string) ([]ForgottenDomain, error) {
```

Comment:
This was moved from internal/server/data/forgotdomain.go.

I also formatted the query across multiple lines so that it is easier to read and changed to use the `scanRows` helper.